wagtail/admin/views/account.py

```diff
@@ -224,43 +227,74 @@ def get_form(self):
 # Views
 
 
-@sensitive_post_parameters()
-def account(request):
-    # Fetch the user and profile objects once and pass into each panel
-    # We need to use the same instances for all forms so they don't overwrite each other
-    user = request.user
-    profile = UserProfile.get_for_user(user)
-
-    # Panels
-    panels = [
-        NameEmailSettingsPanel(request, user, profile),
-        AvatarSettingsPanel(request, user, profile),
-        NotificationsSettingsPanel(request, user, profile),
-        LocaleSettingsPanel(request, user, profile),
-        ThemeSettingsPanel(request, user, profile),
-        ChangePasswordPanel(request, user, profile),
-    ]
-    for fn in hooks.get_hooks("register_account_settings_panel"):
-        panel = fn(request, user, profile)
-        if panel and panel.is_active():
-            panels.append(panel)
-
-    panels = [panel for panel in panels if panel.is_active()]
-
-    # Get tabs and order them
-    tabs = list({panel.tab for panel in panels})
-    tabs.sort(key=lambda tab: tab.order)
-
-    # Get dict of tabs to ordered panels
-    panels_by_tab = OrderedDict([(tab, []) for tab in tabs])
-    for panel in panels:
-        panels_by_tab[panel.tab].append(panel)
-    for tab, tab_panels in panels_by_tab.items():
-        tab_panels.sort(key=lambda panel: panel.order)
-
-    panel_forms = [panel.get_form() for panel in panels]
-
-    if request.method == "POST":
+@method_decorator(sensitive_post_parameters(), name="post")
+class AccountView(WagtailAdminTemplateMixin, TemplateView):
+    template_name = "wagtailadmin/account/account.html"
+
```

Comment:
```suggestion
    page_title = gettext_lazy("Account")
    header_icon = "user"

```

Using the title here means we do not need to include the title in the template

```diff
--{% block titletag %}{% trans "Account" %}{% endblock %}
```

I tried using the built in header in the base template but it looks like we cannot use that yet. FYI @thibaudcolas @laymonage  - the 'merged' header and the 'nice-padding' seems to conflict with the layout of the account view where the tabs should show full width and the header should be slightly smaller. Not sure on an elegant approach here but may need to be considered for the Outreachy.

Observe if I use the `{% block main_content %}` alongside this change, the tabs 'line' is not right and the header is too high.

<img width="1112" alt="Screenshot 2023-11-10 at 5 28 44 pm" src="https://github.com/wagtail/wagtail/assets/1396140/7a7adeb1-1298-4e8d-8584-15c9c0067334">